src/hera/workflows/workflow.py

```diff
@@ -436,5 +444,81 @@ def to_file(self, output_directory: Union[Path, str] = ".", name: str = "", *arg
         output_path.write_text(self.to_yaml(*args, **kwargs))
         return output_path.absolute()
 
+    @classmethod
+    def _from_model(cls: Workflow, model: _ModelWorkflow) -> Workflow:
```

Comment:
This looks great! 

- What do you think about making use of this in the APIs? create, get, lint, etc
- What do you think about making this public?

Reply:
>create, get, lint

Yes was thinking that would be a great use of it! (But would need separate functions for bw-compat.) Makes sense that you would get Hera objects from the workflows service

>making this public

I was leaning towards public but went private because we can go private -> public if we want. Also if the workflows service gives back hera objects then would clients of the library ever need to deal with `_ModelWorkflows`? They would be able to get hera objects from yaml and the live service

Reply:
>I was leaning towards public but went private because we can go private -> public if we want

Sounds great!

Reply:
>the service should primarily use the generated objects

Makes sense to me! 💯 

>interfacing with argo through maintained objects should return maintained objects

Also makes sense, thanks for the example! 💯